sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -297,10 +297,6 @@ func (m *networkFlowManager) enrichAndSendProcesses() {
 
 	m.enrichedProcessesLastSentState = currentProcesses
 
-	if len(updatedProcesses) == 0 {
```

Comment:
Removing this was necessary so that old plops would be retried even when there are no new plops.